gax/src/test/java/com/google/api/gax/rpc/CancellationTest.java

```diff
@@ -70,7 +70,7 @@
           .setInitialRpcTimeout(Duration.ofMillis(2L))
           .setRpcTimeoutMultiplier(1)
           .setMaxRpcTimeout(Duration.ofMillis(2L))
-          .setTotalTimeout(Duration.ofMillis(10L))
+          .setTotalTimeout(Duration.ofMillis(20L))
```

Comment:
Why is this necessary?

Reply:
Bazel build has nice feature of running same tests multiple times to detect flaky tests (with comman line arg `--runs_per_test=<number_of_runs>`). Also, when bazel runs tests it is doing it in parallel, thus consuming more resources. Those tests are time-based and, as results, depend on performance of the machine.  Those changes are needed to reduce potential flakiness of the tests. I was able to run full set of tests with `--runs_per_test=50` without flakiness with these changes.